### DISCUSSION TITLE
Notifications to discord?

### DISCUSSION THREAD

**Discussion by DoAndroids** [cross-seed/cross-seed#322] [Q&A] [answered]
Hi

I cannot get the notifications working as I'd like.
I've attempted simply putting the Webhook URL for notifications, but nothing appears in Discord.
Also tried using the apprise shorthand 'url' (discord://webhookId/webookToken), but that errors.

Can someone point me in the right direction?

Thanks

**Comment by mmgoodnow** [comment, marked as answer]
You have to set up another app: Notifiarr or apprise-api. If using apprise, you'll have to put your discord:// URL into the config for your apprise-api instance. 


https://www.cross-seed.org/docs/reference/options#notificationwebhookurl

**Reply by DoAndroids** [reply]
Thank you, I'll take a look at that
